parsers/IN_WE.py

```diff
@@ -67,7 +69,7 @@ def fetch_data(
 
     datetime_col = KIND_MAPPING[kind]["datetime_column"]
     for item in data:
-        item[datetime_col] = datetime.strptime(item[datetime_col], "%Y-%d-%m %H:%M:%S")
+        item[datetime_col] = datetime.strptime(item[datetime_col], "%Y-%m-%d %H:%M:%S")
         dt = arrow.get(item[datetime_col])
```

Comment:
Since we are changing things in here anyway I would prefer if we could get rid of this usage of arrow here. Python datetimes should be able to do the same thing without much hassle.

Reply:
I will do that in another PR